frontend/catalyst/jit.py

```diff
@@ -433,6 +433,42 @@ def sum_abstracted(arr):
 
         the ``sum_abstracted`` function would only compile once and its definition would be
         reused for subsequent function calls.
+
+    .. details::
+        :title: Qubit-invariant compilation
+
+        A powerful feature of Catalyst is that the same compiled qnode can be evoked with
```

Comment:
I usually see it written as "QNode" (e.g. https://docs.pennylane.ai/en/stable/introduction/circuits.html):

```suggestion
        A powerful feature of Catalyst is that the same compiled QNode can be evoked with
```